.gitignore

```diff
@@ -48,6 +48,9 @@ cmake-build-*
 cuobjdump
 nvdisasm
 ptxas
+cuobjdump.exe
+nvdisasm.exe
+ptxas.exe
```

Comment:
Do you need these given you leave `dst_path` as not having the file extension when you call `download_and_copy`?

Reply:
You are right, it is not necessary, thanks!